docs/content/latest/admin/yb-admin.md

```diff
@@ -677,11 +677,11 @@ yb-admin -master_addresses <master-addresses> delete_snapshot <snapshot_id>
 
 ---
 
-### Deployment topology commands
+<a name="deployment-topology-commands"></a>
```

Comment:
not sure if this is working as expected?

Reply:
It is... click the link "Deployment topology" link in the TOC at the top of the Commands section and you can see it working:

https://deploy-preview-7450--infallible-bardeen-164bc9.netlify.app/latest/admin/yb-admin/#commands
https://deploy-preview-7450--infallible-bardeen-164bc9.netlify.app/latest/admin/yb-admin/#deployment-topology-commands

Reply:
ooooh, makes sense that it takes you straight to `Multi-zone and multi-region`, since we don't have a `Deployment topology` header anymore